### ISSUE TITLE
OData support 

### ISSUE THREAD

**Issue by behzadeshan** [balderdashy/sails#3093]
Do you have any plan to support OData?

**Comment by tjwebb** [comment]
I see this sort of as the same as supporting SOAP, or json-schema, or any other request/response format. Sails can certainly support these things through extensions ([hooks](http://sailsjs.org/documentation/concepts/extending-sails/hooks)). I know that kendo-ui and some other more enterprise-y frameworks are preferring OData, so any support for this via a Sails hook would be fantastic.

**Comment by netrevisanto** [comment]
This would be Awesome.

**Comment by sailsbot** [comment]
Thanks for posting, @behzadeshan.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
